Makefile

```diff
@@ -0,0 +1,35 @@
+.DEFAULT_GOAL = help
+
+REAL_ENV = dev
+NAME = storybook-${REAL_ENV}
+TAG = libero/storybook:${REAL_ENV}
+EXISTING_CONTAINERS = $$(docker ps --all --quiet --filter "name=${NAME}")
+
+help: ## Display this help text
+	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
+
+install: ## Install dependencies locally
+	npm install
+
+build: ## Build the container
+	docker build --target ${REAL_ENV} --tag ${TAG} .
+
+start: ## Start the container
```

Comment:
not idempotent as the name of the container will be in use - could use a dependency on `stop` if the intent is to restart it, or a check otherwise

Reply:
Some of the others aren't either, and can't have a dependency (eg `sh`).

Probably fine for now.